test-bench-utils/lib/sinks/xxe.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
-
-const libxmljs = require('libxmljs');
+let libxmljs;
+try {
+  libxmljs = require('libxmljs');
+} catch (err) {
+  libxmljs = null;
```

Comment:
Pretty sure you don't need this defaulting logic, if `require` throws the assignment will never occur.  So you can just do:

```
try {
  libxmljs = require('libxmljs');
} catch {}
```